docs/ONNXTypes.md

```diff
@@ -19,22 +19,39 @@ Optional type only appears in TorchScript graphs generated by jit script compile
 
     Corresponding TorchScript graph:
 
-        graph(
+        Graph(
             %self : __torch__.Model, 
             %x.1 : Tensor, 
             %y.1 : Tensor?
         ): 
             %11 : int = prim::Constant[value=1]() 
             %4 : None = prim::Constant() 
             %5 : bool = aten::__isnot__(%y.1, %4) 
-            %36 : Tensor = prim::If(%5) 
+            %6 : Tensor = prim::If(%5) 
                 block0(): 
                     %y.4 : Tensor = prim::unchecked_cast(%y.1) 
                     %12 : Tensor = aten::add(%x.1, %y.4, %11) 
                 -> (%12) 
                 block1(): 
                 -> (%x.1) 
-            return (%36) 
+            return (%6)
+
+    ONNX graph:
+
+        Graph(
+            %x.1 : Float(2, 3, strides=[3, 1], requires_grad=0, device=cpu), 
```

Comment:
Can we drop all the extraneous attributes like strides, requires_grad, and device? They are somewhat confusing and misleading in an ONNX context. Same in all lines below as well. Thanks!